src/app/products/loan-products/loan-product-stepper/loan-product-currency-step/loan-product-currency-step.component.html

```
@@ -47,7 +47,12 @@
       </mat-error>
     </mat-form-field>
 
-    <mat-checkbox labelPosition="before" formControlName="setMultiples" class="flex-98 margin-t">
+    <mat-checkbox
+      labelPosition="before"
+      formControlName="setMultiples"
+      class="flex-98 margin-t"
+      style="align-items: center"
```

HTMLHint (1.8.0)

[warning] Inline style [ 
      style="align-items: center" ] cannot be used.

(inline-style-disabled)